packages/wrangler/e2e/helpers/normalize.ts

```diff
@@ -104,7 +104,7 @@ export function stripTimings(stdout: string): string {
 export function npmStripTimings(stdout: string): string {
 	return stdout
 		.replace(
-			/added \d+ packages, and audited \d+ packages in \d+s/,
+			/added \d+ packages, and audited \d+ packages in [\dms]+/,
```

Comment:
`\d+m?s`?

Reply:
If it is really slow it can take more than a minute, in which case you get `...and audited 20 pages in 1m` rather than say `...and audited 20 pages in 60s`